### ISSUE TITLE
model: autosklearn: Container and docs failing due to ConfigSpace numpy inconpatabiltiy

### ISSUE THREAD

**Issue by pdxjohnny** [intel/dffml#992]
```
+ docker run --rm intelotc/dffml version
dffml 0.3.7 /usr/src/dffml/dffml
dffml-config-yaml 0.0.10 /usr/src/dffml/configloader/yaml/dffml_config_yaml
dffml-config-image 0.0.3 /usr/src/dffml/configloader/image/dffml_config_image
dffml-model-scratch 0.0.8 /usr/src/dffml/model/scratch/dffml_model_scratch
dffml-model-scikit 0.0.9 /usr/src/dffml/model/scikit/dffml_model_scikit
dffml-model-tensorflow 0.2.7 /usr/src/dffml/model/tensorflow/dffml_model_tensorflow
dffml-model-tensorflow-hub 0.0.5 /usr/src/dffml/model/tensorflow_hub/dffml_model_tensorflow_hub
dffml-model-vowpalWabbit 0.0.1 /usr/src/dffml/model/vowpalWabbit/dffml_model_vowpalWabbit
dffml-model-xgboost 0.0.1 /usr/src/dffml/model/xgboost/dffml_model_xgboost
dffml-model-pytorch 0.0.1 /usr/src/dffml/model/pytorch/dffml_model_pytorch
dffml-model-spacy 0.0.1 /usr/src/dffml/model/spacy/dffml_model_spacy
dffml-model-daal4py 0.0.1 /usr/src/dffml/model/daal4py/dffml_model_daal4py
Traceback (most recent call last):
  File "/usr/local/bin/dffml", line 33, in <module>
    sys.exit(load_entry_point('dffml', 'console_scripts', 'dffml')())
  File "/usr/src/dffml/dffml/util/cli/cmd.py", line 288, in main
    result = loop.run_until_complete(cls._main(*argv[1:]))
  File "/usr/local/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/usr/src/dffml/dffml/util/cli/cmd.py", line 238, in _main
    result = await cls.cli(*args)
  File "/usr/src/dffml/dffml/util/cli/cmd.py", line 224, in cli
    return await cmd.do_run()
  File "/usr/src/dffml/dffml/util/cli/cmd.py", line 203, in do_run
    return await self.run()
  File "/usr/src/dffml/dffml/cli/cli.py", line 131, in run
    module = importlib.import_module(module_name)
  File "/usr/local/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/src/dffml/model/autosklearn/dffml_model_autosklearn/__init__.py", line 30, in <module>
    from .autoregressor import AutoSklearnRegressorModel
  File "/usr/src/dffml/model/autosklearn/dffml_model_autosklearn/autoregressor.py", line 2, in <module>
    import autosklearn.regression
  File "/usr/local/lib/python3.7/site-packages/autosklearn/regression.py", line 1, in <module>
    from autosklearn.estimators import AutoSklearnRegressor  # noqa (imported but unused)
  File "/usr/local/lib/python3.7/site-packages/autosklearn/estimators.py", line 11, in <module>
    from autosklearn.automl import AutoMLClassifier, AutoMLRegressor, AutoML
  File "/usr/local/lib/python3.7/site-packages/autosklearn/automl.py", line 16, in <module>
    from ConfigSpace.read_and_write import json as cs_json
  File "/usr/local/lib/python3.7/site-packages/ConfigSpace/__init__.py", line 37, in <module>
    from ConfigSpace.configuration_space import Configuration, \
  File "ConfigSpace/configuration_space.pyx", line 39, in init ConfigSpace.configuration_space
  File "ConfigSpace/hyperparameters.pyx", line 1, in init ConfigSpace.hyperparameters
ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 88 from C header, got 80 from PyObject
```

References:
- https://github.com/automl/ConfigSpace/issues/173

**Comment by pdxjohnny** [comment]
We need to track this and remove our temporary fix when https://github.com/automl/ConfigSpace/pull/175 is merged

**Comment by pdxjohnny** [comment]
Looks like they've merged but we're waiting on a verison bump

**Comment by pdxjohnny** [comment]
Fixed in 4d8a10dc19ab562240b8567a3b173fc7f1b90f6a
